data/json/recipes/recipe_appliance.json

```diff
@@ -106,6 +106,38 @@
     "using": [ [ "soldering_standard", 5 ] ],
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "proficiencies": [ { "proficiency": "prof_elec_soldering" } ],
-    "components": [ [ [ "frame_wood", 1 ] ], [ [ "battery_charger", 1 ] ] ]
+    "components": [ [ [ "frame_wood", 1 ] ], [ [ "battery_charger", 1 ] ], [ [ "medium_battery_cell", 1 ] ] ]
```

Comment:
why modify this unrelated recipe?

Reply:
This is a conflict resolution... The unrelated recipe was changed in the meantime. But maybe  that was incorrect way how to fix the conflict?

Reply:
Not sure what you did but this is not right